### PR TITLE
Update minimum software requirements

### PR DESCRIPTION
**Pull request by jasonblais**
For context of this PR, see https://github.com/mattermost/docs/pull/2180

### CODE REVIEW

File: components/error_page/error_message.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default function ErrorMessage({type, message, service}) {
                         <li>
                             <FormattedMessage
                                 id='error.local_storage.help3'
-                                defaultMessage='Use a supported browser (IE 11, Chrome 43+, Firefox 38+, Safari 9, Edge)'
+                                defaultMessage='Use a supported browser (IE 11, Chrome 56+, Firefox 60+, Safari 12+, Edge 42+)'
```
Comment (crspeller) [inline components/error_page/error_message.jsx]: Desktop app is going to be bumped to updated electron. It's now on 66 so might as well bump it here. 
```suggestion
                                defaultMessage='Use a supported browser (IE 11, Chrome 66+, Firefox 60+, Safari 12+, Edge 42+)'
```

File: i18n/en.json [hunk 2]
```diff
@@ -1765,7 +1765,7 @@
   "error.generic.title": "Error",
   "error.local_storage.help1": "Enable cookies",
   "error.local_storage.help2": "Turn off private browsing",
-  "error.local_storage.help3": "Use a supported browser (IE 11, Chrome 43+, Firefox 52+, Safari 9, Edge 40+)",
+  "error.local_storage.help3": "Use a supported browser (IE 11, Chrome 56+, Firefox 60+, Safari 12+, Edge 42+)",
```
Comment (crspeller) [inline i18n/en.json]:
```suggestion
  "error.local_storage.help3": "Use a supported browser (IE 11, Chrome 66+, Firefox 60+, Safari 12+, Edge 42+)",
```

File: tests/components/error_page/__snapshots__/error_message.test.jsx.snap [hunk 3]
```diff
@@ -34,7 +34,7 @@ exports[`components/error_page/ErrorMessage should match snapshot, local_storage
     </li>
     <li>
       <FormattedMessage
-        defaultMessage="Use a supported browser (IE 11, Chrome 43+, Firefox 38+, Safari 9, Edge)"
+        defaultMessage="Use a supported browser (IE 11, Chrome 56+, Firefox 60+, Safari 12+, Edge 42+)"
```
Comment (crspeller) [inline tests/components/error_page/__snapshots__/error_message.test.jsx.snap]:
```suggestion
        defaultMessage="Use a supported browser (IE 11, Chrome 66+, Firefox 60+, Safari 12+, Edge 42+)"
```

File: components/error_page/error_message.jsx (hunk 1 above)
Comment (jasonblais) [reply components/error_page/error_message.jsx]: Thanks @crspeller, makes sense. Looks like the desktop app master branch is running an Electron version with Chrome v61 though I think there's a plan to bump the Electron version up before the release. 

But I'll wait for the desktop app release and update this PR afterwards with the Chrome version we're using.

**Comment by jasonblais** [comment]
@crspeller Are you okay merging this PR as-is (apart from a rebase)?

The most recent desktop app version (v4.1.2) uses Chrome 56, and I'm not sure when v4.2.0 is going to ship. Looks like the auto-updater PR is being actively discussed https://github.com/mattermost/desktop/pull/582

v4.2.0 will have Chrome 61 (via Electron v2), v4.3.0 will have Chrome 66 (via Electron v3). Our feature release process has a step to update these requirements, so we can cover changes with that.

**Comment by crspeller** [comment]
@jasonblais Ya I didn't realize we where holing back an electron version. This PR should at least be updated to Chrome 61 as the next desktop release should happen soon.

**Pull request closed without merging**